File: TeamWorld/App/models/dto/requests/registration-request.ts
import { BaseRequest } from '../base/base-request';
import { RegistrationData } from '../../interface/registration-data';

export class RegistrationRequest {
    login: string;
    name: string;
    surName: string;
    email: string;
    passpord: string;
    phone: string;

    constructor(registrationData: RegistrationData) {
        this.login = registrationData.login;
        this.name = registrationData.name;
        this.surName = registrationData.surName;
        this.email = registrationData.email;
        this.passpord = registrationData.passpord;
        this.phone = registrationData.phone;
    }
}

File: TeamWorld/Models/Requests/RegistrationRequest.cs
namespace TeamWorld.Models.Requests
{
    public class RegistrationRequest
    {
        public string Login { set; get; }

        public string Name { set; get; }

        public string SurName { set; get; }

        public string Email { set; get; }

        public string Passpord { set; get; }

        public string Phone { set; get; }
    }
}